README.md

```diff
@@ -35,6 +35,24 @@ $ pre-commit install
 $ streamlit run streamlit_agent/mrkl_demo.py
 ```
 
+# Running with Docker
+
+This project includes `Dockerfile` to run the app in Docker container. In order to optimise the Docker Image is optimsed for size and building time with cache techniques.
+
+To generate Image with `DOCKER_BUILDKIT`, follow below command
```

Comment:
Please add instructions on how to launch the image with a different app besides chat_pandas_df